### DISCUSSION TITLE
Nicotine+ should be a cask, not a formula.

### DISCUSSION THREAD

**Discussion by Red6785** [Homebrew/discussions#5829] [Casks] [answered]
### Output of `brew config`

```shell
red@binah ~ % brew config
HOMEBREW_VERSION: 4.4.13
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6ea9df68df4868ce12ea7c51c85b57dce514f2e8
Last commit: 3 days ago
Branch: stable
Core tap JSON: 25 Dec 19:01 UTC
Core cask tap JSON: 25 Dec 19:01 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 14
HOMEBREW_NO_EMOJI: set
HOMEBREW_NO_ENV_HINTS: set
Homebrew Ruby: 3.3.6 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.6/bin/ruby
CPU: 14-core 64-bit arm_brava
Clang: 16.0.0 build 1600
Git: 2.47.1 => /opt/homebrew/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.2-arm64
CLT: 16.2.0.0.1.1733547573
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
red@binah ~ % brew doctor
Your system is ready to brew.
```

### Description of issue

Shouldn't Nicotine+ be a cask, instead of a formula? It is a GUI-based app. It is not intended to be ran from the terminal - interaction happens purely within the GUI. It is installed from the official release by dragging it into /Applications and running that.
brew install nicotine-plus installs a formula that has to be run with ./nicotine in the terminal.
brew install --cask nicotine-plus returns nothing.

### Relevant casks

none; that's the issue

**Comment by p-linnane** [comment]
Having a GUI doesn't mean that something needs to be a cask. We package `gtk4` and `qt` for building formulae with GUIs where applicable.

**Reply by Red6785** [reply]
While that is true, most graphical software is put in casks due to it being much easier to use, even the ones that are easily to build.

**Reply by SMillerDev** [reply]
Casks have the disadvantage of not working on Linux though, so there is a balance to be struck there.

**Comment by Red6785** [comment, marked as answer]
It's because Nicotine+ isn't signed. I'm sorry for wasting time.
